material/typography.html

```diff
@@ -0,0 +1,81 @@
+<custom-style>
+  <style>
+    html {
+      --material-font-family: 'Roboto', 'Noto', sans-serif;
+      --material-font-size: 16px;
+
+      /* TODO: should probably use Material design naming instead */
+      --material-font-scale-up: 1.25;
+      --material-font-scale-down: 1.14;
+
+      --material-font-size-xxxl: calc(var(--material-font-size) * var(--material-font-scale-up) * var(--material-font-scale-up) * var(--material-font-scale-up) * var(--material-font-scale-up));
+      --material-font-size-xxl: calc(var(--material-font-size) * var(--material-font-scale-up) * var(--material-font-scale-up) * var(--material-font-scale-up));
+      --material-font-size-xl: calc(var(--material-font-size) * var(--material-font-scale-up) * var(--material-font-scale-up));
+      --material-font-size-l: calc(var(--material-font-size) * var(--material-font-scale-up));
+      --material-font-size-m: var(--material-font-size);
+      --material-font-size-s: calc(var(--material-font-size) / var(--material-font-scale-down));
+      --material-font-size-xs: calc(var(--material-font-size) / var(--material-font-scale-down) / var(--material-font-scale-down));
+
+      --material-font-weight-strong: 500;
```

Comment:
Related to strong font in Linux. Should this be `700`? https://css-tricks.com/almanac/properties/f/font-weight/